src/runtime/storage/pouchdb/pouch-db-variable.ts

```diff
@@ -288,7 +288,8 @@ export class PouchDbVariable extends PouchDbStorageProvider implements VariableS
    */
   private async getStored(): Promise<ValueStorage> {
     try {
-      const result = await this.db.get(this.pouchDbKey.location);
+      // tslint:disable-next-line: no-any
+      const result: any = await this.db.get(this.pouchDbKey.location);
```

Comment:
We do try to avoid using `any` unless it's genuinely required. Can this be typed any more narrowly?

Reply:
this.db.get returns a result of type `PouchDB.Core.IdMeta & PouchDB.Core.GetMeta`. There's a bunch of properties below like `result.referenceMode` that don't exist on those types. What type should this be instead?

Reply:
This would also be handled by defining an interface covering the used fields.

Reply:
Done.